com.unity.film-tv.toolbox/Samples/MultiScene.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
You are missing .samples.json. https://github.cds.internal.unity3d.com/unity/com.unity.package-starter-kit/blob/master/Samples/Example/.sample.json